examples/3D-pol-to-birefringence.py

```diff
@@ -78,24 +78,15 @@
 ## Save to zarr
 # Save birefringence
 writer = WaveorderWriter("./output")
-writer.create_zarr_root("birefringence_" + timestamp)
+writer.create_zarr_root("reconstructions_" + timestamp)
 writer.init_array(
     position=0,
-    data_shape=(1, 4, Z, Y, X),
+    data_shape=(1, 5, Z, Y, X),
     chunk_size=(1, 1, 1, Y, X),
-    chan_names=["Retardance", "Orientation", "BF", "Pol"],
+    chan_names=["Retardance", "Orientation", "BF", "Pol", "Phase"],
```

Comment:
I think we should rename `Pol` to `DoP` for degree of polarization

Reply:
Thanks! 

I also changed the name of "BF" to "S0" since that's more description IMO.

Reply:
Previously we used something called `computed brightfield` for this quantity. I think it's not immediately intuitive that S0 equated to brightfield. I personally think calling it `BF` is good enough for the purpose of a channel name

Reply:
>> I think it's not immediately intuitive that S0 equated to brightfield

Do we want users to equate S0 to brightfield? I think some distinction between S0 and brightfield is valuable since S0 depends on details of the LCs, calibration, reconstruction, etc.  Maybe `BF-S0` or `BF-pol` or `BF-computed` are okay compromises?

I'm not extremely attached to the distinction, so I'll yield to your final call @ieivanov.

Reply:
Just returning to this discussion. I think something like `BF - computed` should do